libtransmission/utils.cc

```diff
@@ -33,10 +33,6 @@
 #include <sys/stat.h> // mode_t
 #endif
 
-#ifdef HAVE_ICONV
-#include <iconv.h>
```

Comment:
`#include <tuple>` above could be removed too.

Reply:
Removed in a62195e